File: backend/src/main/java/ca/metricalsky/winston/repository/SettingsRepository.java
package ca.metricalsky.winston.repository;

import ca.metricalsky.winston.entity.SettingsEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SettingsRepository extends JpaRepository<SettingsEntity, String> {

}

File: database/src/main/kotlin/ca/metricalsky/winston/database/repository/SettingsRepository.kt
package ca.metricalsky.winston.database.repository

import ca.metricalsky.winston.database.entity.SettingsEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SettingsRepository: JpaRepository<SettingsEntity, String>
